src/less/techne-patterns/_trees.less

```diff
@@ -1,3 +1,401 @@
+// new tree
+//
+// this is the new tree pattern
+//
+// Markup:
+// <div class="y-tree">
+//     <!-- headers -->
+//     <div class="row tablehead hidden-sm hidden-xs">
+//         <div class="col-sm-4">
+//             <div class="col-sm-1 ">    
+//                 <a class="btn btn-link category-toggle ">
+//                     <span class="glyphicon glyphicon-chevron-down" ></span>
+//                 </a>
+//             </div>
+//             <div class="col-sm-10">Data Header</div>
+//         </div>
+//         <div class="col-sm-3 ng-binding">Data Header</div>
+//         <div class="col-sm-3 ng-binding">Data Header</div>
+//     </div>
+//     <div class="row tablehead visible-sm visible-xs">
+//         <div class="col-xs-1 ">    
+//             <a class="btn btn-link category-toggle ">
+//                 <span class="glyphicon glyphicon-chevron-down" ></span>
+//             </a>
+//         </div>
+//         <div class="col-xs-8 ng-binding">Data Header</div>
+//     </div>
+//     <div class="divider"></div>
+//     <!-- nodes -->
+//     <!-- node 1 -->
+//     <div class="row mobileLayout visible-xs visible-sm ">
```

Comment:
do we really want to change all the existing class names (in common with the old tree) and force everyone to do a full markup update?